src/python/pants/releases/packages.py

```diff
@@ -182,7 +182,7 @@ def build_and_print_packages(version):
     packages_by_flags[package.bdist_wheel_flags].append(package)
 
   for (flags, packages) in packages_by_flags.items():
-    args = ("./pants", "-q", "setup-py", "--run=bdist_wheel {}".format(" ".join(flags))) + tuple(package.target for package in packages)
+    args = ("./pants.pex", "-q", "setup-py", "--run=bdist_wheel {}".format(" ".join(flags))) + tuple(package.target for package in packages)
```

Comment:
This kills a local run of `./build-support/bin/release.sh -n`, but that is no longer par of running a release locally so maybe OK, but it is ugly:
```
$ ./build-support/bin/release.sh -n
[=== 00:00 Performing a dry run release ===]
[=== 00:00 Building packages ===]
Traceback (most recent call last):
  File "/tmp/tmpSVf9la/.bootstrap/_pex/pex.py", line 349, in execute
    exit_code = self._wrap_coverage(self._wrap_profiling, self._execute)
  File "/tmp/tmpSVf9la/.bootstrap/_pex/pex.py", line 281, in _wrap_coverage
    return runner(*args)
  File "/tmp/tmpSVf9la/.bootstrap/_pex/pex.py", line 312, in _wrap_profiling
    return runner(*args)
  File "/tmp/tmpSVf9la/.bootstrap/_pex/pex.py", line 397, in _execute
    return self.execute_interpreter()
  File "/tmp/tmpSVf9la/.bootstrap/_pex/pex.py", line 456, in execute_interpreter
    self.execute_content(arg, content)
  File "/tmp/tmpSVf9la/.bootstrap/_pex/pex.py", line 491, in execute_content
    exec_function(ast, globals())
  File "<exec_function>", line 4, in exec_function
  File "/home/jsirois/dev/pantsbuild/jsirois-pants/src/python/pants/releases/packages.py", line 260, in <module>
    build_and_print_packages(sys.argv[2])
  File "/home/jsirois/dev/pantsbuild/jsirois-pants/src/python/pants/releases/packages.py", line 187, in build_and_print_packages
    subprocess.check_call(args)
  File "/tmp/tmpSVf9la/.deps/subprocess32-3.2.7-cp27-cp27mu-linux_x86_64.whl/subprocess32.py", line 597, in check_call
    retcode = call(*popenargs, **kwargs)
  File "/tmp/tmpSVf9la/.deps/subprocess32-3.2.7-cp27-cp27mu-linux_x86_64.whl/subprocess32.py", line 578, in call
    p = Popen(*popenargs, **kwargs)
  File "/tmp/tmpSVf9la/.deps/subprocess32-3.2.7-cp27-cp27mu-linux_x86_64.whl/subprocess32.py", line 825, in __init__
    restore_signals, start_new_session)
  File "/tmp/tmpSVf9la/.deps/subprocess32-3.2.7-cp27-cp27mu-linux_x86_64.whl/subprocess32.py", line 1574, in _execute_child
    raise child_exception_type(errno_num, err_msg)
OSError: [Errno 2] No such file or directory: u'./pants.pex'
...
```

Perhaps one to add to the cleanup passes.

Reply:
Hmm, I'm not actually sure why this was necessary, since these shards set RUN_PANTS_FROM_PEX=1. I'll revert to calling `./pants` and see what happens.

Reply:
Yep, that appears not to be necessary, reverted that bit, so now `release.sh -n` should work from sources.